<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_change</class>
 <widget class="QMainWindow" name="admin_change">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MOTINUE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_enter">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_exit">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_add">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_change">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>90</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>240</width>
      <height>163</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_name"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>账号</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_number"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>身高</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_class"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>体重</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_time"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>腰围</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_GPA"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_passowrd"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>496</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
